--- a/package/luci-app-mosdns/root/usr/share/mosdns/mosdns.sh
+++ b/package/luci-app-mosdns/root/usr/share/mosdns/mosdns.sh
@@ -106,7 +106,7 @@ geodat_update() (
     TMPDIR=$(mktemp -d) || exit 1
     [ -n "$(uci -q get mosdns.config.github_proxy)" ] && mirror="$(uci -q get mosdns.config.github_proxy)/"
     # geoip.dat - cn-private
-    geoip_type=$(uci -q get mosdns.config.geoip_type || echo "geoip-only-cn-private")
+    geoip_type=$(uci -q get mosdns.config.geoip_type || echo "geoip")
     echo -e "Downloading "$mirror"https://github.com/Loyalsoldier/geoip/releases/latest/download/"$geoip_type".dat"
     curl --connect-timeout 5 -m 120 --ipv4 -kfSLo "$TMPDIR/geoip.dat" ""$mirror"https://github.com/Loyalsoldier/geoip/releases/latest/download/"$geoip_type".dat"
     [ $? -ne 0 ] && rm -rf "$TMPDIR" && exit 1

--- a/package/v2ray-geodata/Makefile
+++ b/package/v2ray-geodata/Makefile
@@ -37,7 +37,7 @@ define Package/v2ray-geosite
   LICENSE:=GPL-3.0
 endef
 
-GEOIP_URL:=https://github.com/Loyalsoldier/geoip/releases/latest/download/geoip-only-cn-private.dat
+GEOIP_URL:=https://github.com/Loyalsoldier/geoip/releases/latest/download/geoip.dat
 GEOSITE_URL:=https://github.com/Loyalsoldier/v2ray-rules-dat/releases/latest/download/geosite.dat
 
 define Build/Compile
