diff --git a/feeds/luci/applications/luci-app-minieap/luasrc/model/cbi/minieap/general.lua b/feeds/luci/applications/luci-app-minieap/luasrc/model/cbi/minieap/general.lua
index 2695e93..76ffa15 100644
--- a/feeds/luci/applications/luci-app-minieap/luasrc/model/cbi/minieap/general.lua
+++ b/feeds/luci/applications/luci-app-minieap/luasrc/model/cbi/minieap/general.lua
@@ -170,6 +170,9 @@ rj_option = o:taboption("ruijie", DynamicList, "rj_option", translate("Custom EA
 rj_option.description = translate("Format &lt;type&gt;:&lt;value&gt;[:r]. Add a option type: &lt;type&gt;, value: &lt;value&gt;. :r for replace")
 
 if nixio.fs.mkdir("/etc/minieap.conf.d") == true then
+    if nixio.fs.stat("/etc/minieap.conf") then
+        nixio.fs.unlink("/etc/minieap.conf")
+    end
 	nixio.fs.symlink("/etc/minieap.conf.d/minieap.conf.utf8", "/etc/minieap.conf")
 end
 
